addons/website/static/src/js/menu/edit.js

```diff
@@ -415,6 +415,7 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
             isRootEditable: false,
             controlHistoryFromDocument: true,
             getContentEditableAreas: this._getContentEditableAreas.bind(this),
+            isHintBlacklisted: node => node.classList && node.classList.contains('nav-item'),
```

Comment:
Not a bad idea to make the fix specific to website navbar but targeting nav-item only is something that could be done in web_editor (which is aware that bootstrap exists and even more) cc @younn-o 

@bso-odoo So we have to choose: either we blacklist all nav-item and it should be done in web_editor IMO or we blacklist our website navbar and do it in website in that case of course.

Also can you validate this with the editor team by pinging them? Thanks :+1: